plugins/inputs/execd/execd_test.go

```diff
@@ -26,6 +26,7 @@ func TestSettingConfigWorks(t *testing.T) {
 	[[inputs.execd]]
 		command = ["a", "b", "c"]
 		restart_delay = "1m"
+		shutdown_timeout = "5s"
```

Comment:
nit: can you do a value that isn't the default to make sure it isn't just using the default?